modules/admin-bar/assets/js/frontend/module.js

```diff
@@ -134,4 +134,4 @@ class AdminBar extends elementorModules.ViewModule {
 	}
 }
 
-new AdminBar();
+document.addEventListener( 'DOMContentLoaded', () => new AdminBar() );
```

Comment:
```suggestion
jQuery( () => new AdminBar() );
```

Reply:
@kobizz see in the code comment.

Reply:
jQuery relies on `DOMContentLoaded` event